docs-ref-mapping/reference-latest.yml

```diff
@@ -499,6 +499,11 @@
       landingPageType: Service
       children:
       - com.azure.resourcemanager.keyvault*
+    - name: Jca
+      uid: azure.java.sdk.landingpage.services.keyvault.Jca
+      href: ~/docs-ref-services/security-keyvault-jca-readme-pre.md
```

Comment:
It doesn't look right to have pre in latest. 

Please make changes to reference-unified.yml and run script to update the latest and preview reference files.

Reply:
Hi @sima-zhu , I remember in our last discussion you told me that the script is triggered by a pipeline automatically and it will update _reference-latest.yml_ according to _reference-unified.yml_. So now I just restored all the changes on _reference-latest.yml_ and removed _-pre_ suffix in reference-unified.yml, is it OK?

Reply:
Yes, that's correct.